src/index.js

```diff
@@ -0,0 +1,44 @@
+/** Copyright (c) 2018 Uber Technologies, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {StoreEnhancer} from 'redux';
+import {createPlugin, createToken} from 'fusion-core';
+import type {Token} from 'fusion-core';
+import {
+  routerMiddleware as createRouterMiddleware,
+  connectRouter,
+} from 'connected-react-router';
+import {RouterToken} from 'fusion-plugin-react-router';
+
+export const ConnectedRouterEnhancerToken: Token<
+  StoreEnhancer<*, *, *>
```

Comment:
Probably should migrate away from the existential type at some point, but I suppose this is fine for now since we have it in other plugins.